index.html

```diff
@@ -16,117 +16,133 @@
           or sign-ups required. has been the industry's standard</p>
-          <div class="see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
           <ul class="languages-work">
             <li class="html-box"><p class="code-name">html</li>
             <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
             <li class="ruby-box"><p class="code-name">Ruby</p></li>
           </ul>
-          
+        </div>
+          <div class="see-three-box see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
        </div>
-
+       <div class="worksec worktwo">
+        <div class="work-box-three work-box-two">
+          <h5 class="work-three-title work-two-title">Multi-Post Stories</h5>
+          <p class="work-three-dis work-two-dis">A daily selection of privately personalized reads; no accounts 
+            or sign-ups required. has been the industry's standard</p>
+            <ul class="languages-work">
+              <li class="html-box"><p class="code-name">html</li>
+              <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+              <li class="ruby-box"><p class="code-name">Ruby</p></li>
+            </ul>
+          </div>
+          <div class="see-three-box see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
+         </div>
+         <div class="workthree worktwo">
+          <div class="work-box-three work-box-two">
+          <h5 class="work-three-title work-two-title">Data Dashboard Healthcare</h5>
+          <p class="work-three-dis work-two-dis">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="languages-work">
+              <li class="html-box"><p class="code-name">html</li>
+              <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+              <li class="ruby-box"><p class="code-name">Ruby</p></li>
+            </ul>
+          </div>
+            <div class="see-three-box see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
+         </div>
+         <div class="workfour worktwo">
+          <div class="work-box-three work-box-two">
+          <h5 class="work-three-title work-two-title">Website Protfolio</h5>
+          <p class="work-three-dis work-two-dis">A daily selection of privately personalized reads; no accounts 
+            or sign-ups required. has been the industry's standard</p>
+            <ul class="languages-work">
+              <li class="html-box"><p class="code-name">html</li>
+              <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+              <li class="ruby-box"><p class="code-name">Ruby</p></li>
+            </ul>
+          </div>
+            <div class="see-three-box see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
```

Comment:
[ ] kindly add a hover effect to all your buttons as it's mentioned on the Figma template 🤓.
Your buttons 👀:  
![hisbutton](https://user-images.githubusercontent.com/62156607/193045124-8decd6ca-e78e-42a9-be38-f8860d77fe2b.PNG)


the Figma buttons 👀:  
![hoverbuuton](https://user-images.githubusercontent.com/62156607/193045012-87f4bbf1-3a8a-45d8-bd6f-ee8612c7a7e9.PNG)